tools/gff2JSON/gff-to-json.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="gff-to-JSON_wrapper" name="GFF to JSON" version="0.1.1">
+    <description>
+        Generate a JSON formated Gene file from GFF format
+    </description>
+    <command>
+        <![CDATA[
+        perl $__tool_directory__/gff-to-json.pl 
+        ${genome1}:$inputFile 
+        #for $q in $queries
+            ${q.genome2}:${q.input2}
```

Comment:
You need to always quote with `"` the inputs of type "text" (can be used for XSS attacks). In this case you can probably quote in this way:

```
"${q.genome2}:${q.input2}"
```